infra/version_check.py

```diff
@@ -13,11 +17,11 @@
         ["git", "describe", "--abbrev=0", "--tags"], check=True, capture_output=True
     )
 except subprocess.CalledProcessError as e:
-    print(f"{e.cmd} failed with return code {e.returncode}")
-    print("stdout:")
-    print(e.stdout)
-    print("stderr:")
-    print(e.stderr)
+    logger.info("%s failed with return code %d", e.cmd, e.returncode)
+    logger.info("stdout:")
+    logger.info(e.stdout)
+    logger.info("stderr:")
+    logger.info(e.stderr)
```

Comment:
This part of the code should be refactored; please keep it as `print()` for now (drop all changes in this file).